google/cloud/datastore/client.py

```diff
@@ -122,7 +122,7 @@ def _extended_lookup(
     missing=None,
```

Comment:
Just so I understand the architecture better. Is this function only called by read requests get, runQuery and runAggregationQuery?

Reply:
done